overlord/snapstate/handlers.go

```diff
@@ -1081,7 +1081,7 @@ func monitoringAbort(st *state.State, snapName string) context.CancelFunc {
 	return aborts[snapName]
 }
 
-func isSnapMonitored(st *state.State, snapName string) bool {
+func IsSnapMonitored(st *state.State, snapName string) bool {
```

Comment:
this needs a doc comment, also should be moved to refreshhints.go or autorefresh.go or snapstate.go, which ever has most of the relevant code?

Reply:
thanks, updated